homeassistant/components/airzone/__init__.py

```diff
@@ -44,6 +48,42 @@ def get_airzone_value(self, key) -> Any:
         raise NotImplementedError()
 
 
+class AirzoneSystemEntity(AirzoneEntity):
```

Comment:
Please move this to an `entity.py`

Reply:
Moved in d8771ce4f2a9eaa4674dc796947a61f61abc2fce